### PR TITLE
Doc: Fix missing negation.

### PR DESCRIPTION
**Pull request by JulienPalard**
Reported by Hug Capella on docs@.

Automerge-Triggered-By: @matrixise

### CODE REVIEW

**Review by aeros — CHANGES_REQUESTED** [review]
As a minor phrasing improvement suggestion, I would recommend changing it from:

> ...  If the :const:`LOCALE` flag is used, matches characters considered alphanumeric in the current locale and the underscore.

to:
> ...  If the :const:`LOCALE` flag is used, matches characters which are considered alphanumeric in the current locale and the underscore.

**Review by eamanu — APPROVED** [review]
LGTM

**Comment by JulienPalard** [comment]
@aeros167 So you mean you're suggestion is to change:

> If the	:const:`LOCALE` flag is
   used, matches characters neither considered alphanumeric in the current locale
   nor the underscore.

to:

> If the	:const:`LOCALE` flag is
   used, matches characters which are neither considered alphanumeric in the current locale
   nor the underscore.

?

**Comment by aeros167** [comment]
@JulienPalard I'll submit the actual suggestion instead of just leaving it as a comment so it's easier to see

File: Doc/library/re.rst [hunk 1]
```diff
@@ -540,8 +540,8 @@ character ``'$'``.
    Matches any character which is not a word character. This is
    the opposite of ``\w``. If the :const:`ASCII` flag is used this
    becomes the equivalent of ``[^a-zA-Z0-9_]``.  If the :const:`LOCALE` flag is
-   used, matches characters considered alphanumeric in the current locale
-   and the underscore.
+   used, matches characters neither considered alphanumeric in the current locale
```
Comment (aeros) [inline Doc/library/re.rst]:
```suggestion
   used, matches characters which are neither alphanumeric in the current locale
```
Also removed "considered", unnecessary filler word in this case.

**Review by aeros — CHANGES_REQUESTED** [review]

**Review by matrixise — APPROVED** [review]
Thank you for your contribution

**Pull request merged**